index.html

```diff
@@ -0,0 +1,87 @@
+<html>
+
+<head>
+  <title>SQL tests</title>
+  <script src="packages/auth/dist/auth.umd.js"></script>
+  <script src="packages/sql/dist/sql.umd.js"></script>
```

Comment:
If this example is using both packages together, should it be inside the 'toolkit' folder (like other index.html examples)?

Reply:
IMHO, not really. The `toolkit` folder should use the `toolkit` package which will hide all of these packages behind a common interface.

This is just a smoke test to be honest.